swagger/api.yaml

```diff
@@ -72,6 +72,45 @@ paths:
   "/enterprise/v3/enterprise-customer/{uuid}/learner-summary":
     $ref: "https://raw.githubusercontent.com/edx/edx-enterprise-data/eb793cf/api.yaml#/endpoints/v2/enterpriseCustomerLearnerSummary"
 
+  # Added by TNL team to support Studio content API
+  "api/contentstore/v1/file_asset/{proxy+}":
```

Comment:
I like that you put the new routes under the /contentstore prefix. I thought about it and believe it's really the right call to keep things relating to our new studio content API closely together.